Makefile

```diff
@@ -171,11 +176,23 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | $(KUBECTL) apply -f -
-
+	
+         
 .PHONY: undeploy
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | $(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: deploy-service
```

Comment:
Why a separate deployment? Shouldn't this be all part of the one operator deployment configuration, but presumably with additional containers?

Reply:
@donpenney  thanks for the comments. i was playing a little bit with the original files and some changes has been done  by mistake. fixing them all and also put one target to deploy the oeprator and the services.